frontend-react/src/components/Admin/OrgsTable.tsx

```diff
@@ -37,7 +37,10 @@ export function OrgsTable() {
         let payload: Partial<MembershipSettings> = {
             parsedName: orgName,
         };
-        if (memberType === MemberType.SENDER) {
+        if (
+            memberType === MemberType.SENDER ||
+            memberType === MemberType.PRIME_ADMIN
+        ) {
```

Comment:
@bgantick this is what i mean. we need to set the `default` sender when switching to an admin role on the settings page... if that makes sense, haha

Reply:
now to use this feature, the user has to choose "admin" in the permissions drop down on the admin org settings page